src/styles/block/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  background: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  margin: auto;
+  padding-bottom: 40px;
+
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 14px;
+  margin-bottom: 24px;
+}
+
+.card__reviews-total {
+  font-size: 10px;
+  display: flex;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 18px;
+  margin-bottom: 16px;
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__price-name {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 164px;
+  height: 38px;
+
+  background: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  transition: border-color 0.4s
+    color 0.4s,
+    background-color 0.4s;
```

Comment:
transition is not working, unfortunately, you can make it in such format and it will work correctly:
transition: ease 0.3s;